test/integration/targets/iam_password_policy/aliases

```diff
@@ -1,2 +0,0 @@
-cloud/aws
```

Comment:
Instead of removing these, please add them to the group "unsupported" to denote that we haven't given CI the proper permissions to run them yet.

Reply:
Yup, understood.  I have fixed this by re-adding the tests and including them in the unsupported group.